### ISSUE TITLE
Command to create states in the app

### ISSUE THREAD

**Issue by felquis** [IonicBrazil/ionic-garden#22]
So guys, I think it is good to use gulp to do this, not yeoman as someone recommended. Ionic CLI integrate some tasks with gulp, I think using gulp tasks is a good way to stay close of Ionic CLI, and it may resolves to some good integrations with Ionic CLI

For example, today if you run `ionic serve` it trigger `gulp watch`.

I imagine something like:
```sh 
gulp create-state stateName
```

It should create these files
```sh
www/states/stateName/
www/states/stateName/route.js
www/states/stateName/controller.js
www/states/stateName/template.html
``` 

Create a state with no Controller
```sh
gulp create-state stateName --ignore-controller
```
output
```sh
www/states/stateName/
www/states/stateName/route.js
www/states/stateName/controller.js
www/states/stateName/template.html
``` 

Short version
```sh
gulp cs stateName -c
```

It should use the conventions listed in this [Style Guide](https://github.com/johnpapa/angular-styleguide#iife)

## Note

Please, send suggestions of commands name, it should be intuitive and no obstructive.

**Comment by giorgiofellipe** [comment]
It would be great!

But, as we are trying to follow conventions what do you think about using these naming conventions? https://github.com/johnpapa/angular-styleguide#naming

**Comment by felquis** [comment]
You're right! :) 

This is mentioned in the [README](https://github.com/IonicBrazil/ionic-garden#follow-the-style-guide) Style Guide session.

Ionic Garden will use AngularJS `.jscsrc` file + Johnpapa's Style Guide.

**Comment by felquis** [comment]
@giorgiofellipe BTW, check out the label [style-guide](https://github.com/IonicBrazil/ionic-garden/labels/style-guide).

Anything you see that isn't following the Style Guide, please send a pull request or create an issue explaining what is wrong and what is good.
